.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -38,8 +38,9 @@ body:
         - OS Version: [e.g. Ubuntu 24.04.2 LTS]
         - PostgreSQL Version: [e.g. 15.1]
         - PGroonga Version: [e.g. 12.1.0]
+           - SELECT extversion FROM pg_catalog.pg_extension WHERE extname = 'pgroonga';
```

What do you think of adding the comment about this query although it might be enough?
```suggestion
           - (You can check the version by running: SELECT extversion FROM pg_catalog.pg_extension WHERE extname = 'pgroonga';)
```

---

I added comment.